docs/source/api.rst

```diff
@@ -253,6 +253,33 @@ PDC object APIs
 	* Delete data from an object.
 	* For developers: see pdc_client_connect.c. Use PDC_obj_get_info to retrieve name. Then forward name to servers to fulfill requests.
 
+---------------------------
+PDC region APIs
+---------------------------
+
+
+---------------------------
+PDC property APIs
+---------------------------
+
```

Comment:
Do you know if we have any documentation to include here? Otherwise, we can just remove this.